### ISSUE TITLE
[a11y] Skip to content helper needs to be first focusable item in body, and have accurate link

### ISSUE THREAD

**Issue by jaredhenderson** [unimelb/pattern-lib#1653]
## Currently

The 'skip to content' component is currently located as a child of the first `div` within the `body`. There are several SVG elements injected before this (spritesheets). 

> The link should be the first focusable control on the web page and be visible when it receives focus

It also needs to jump to the correct section of the page. Currently this is defaulting to `#main`, but we don't have any elements that have an [`id` of main](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#document_fragments)

![Screen Shot 2021-06-03 at 3.46.01 pm.png](https://images.zenhubusercontent.com/5acae0cd4b5806bc2bce3338/fc0c23fc-6e83-41bc-bb07-cf3726db2289)

## Proposed change
- [x] Validate what is being focused first on this page - is it the SVG sprite elements? If so, remove these from taborder + screenreaders
- [x] Adjust screenreader component implementation to point to an accurate **ID**
- [x] Can SVG sprites be injected at the end of the body instead of the start? Seems like a lot more can go wrong having them as the first thing than the last. Not sure if they need to be 'defined' [early in the page](https://css-tricks.com/svg-sprites-use-better-icon-fonts/#inject-that-svg-at-the-top-of-the-document) though... looks like that was only a requirement back in 2014

**Comment by gepifanio** [comment]
pr: https://github.com/unimelb/pattern-lib/pull/1656
